src/gwt/src/org/rstudio/studio/client/workbench/views/files/Files.java

```diff
@@ -525,15 +525,11 @@ void onRenameFile()
       ArrayList<FileSystemItem> selectedFiles = view_.getSelectedFiles();
 
       // validation: some selection exists
-      if  (selectedFiles.size() == 0)
```

Comment:
In the interest of reducing risk for Ghost Orchid, I'd recommend not making any changes in GWT that aren't strictly needed by this Electron tweak, so I'd leave out these tweaks to Files.java.

Unlikely to cause any problems but it is a slight change in behavior which could break an integration test (didn't look to see if we have one for this code), generating more noise.

Reply:
I'm being overly nitpicky, merging!

Reply:
For context, I decided to make this change after seeing that clicking the Rename button without anything selected just silently did nothing, which is a somewhat confusing UI experience. This is marginally better, I think?